test/go/iterative_verifier_integration_test.go

```diff
@@ -1,293 +0,0 @@
-	ran := false
-
-	testcase := &testhelpers.IntegrationTestCase{
-		T:           t,
-		SetupAction: setupSingleTableDatabase,
-		AfterRowCopyIsComplete: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			setupIterativeVerifierFromFerry(iterativeVerifier, ferry.Ferry)
-
-			err := iterativeVerifier.Initialize()
-			testhelpers.PanicIfError(err)
-
-			err = iterativeVerifier.VerifyBeforeCutover()
-			testhelpers.PanicIfError(err)
-		},
-		BeforeStoppingBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			ensureTestRowsAreReverified(ferry)
-		},
-		AfterStoppedBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			modifyDataColumnInSourceDB(ferry)
-			result, err := iterativeVerifier.VerifyDuringCutover()
-			assert.Nil(t, err)
-			assert.False(t, result.DataCorrect)
-			assert.Regexp(t, "verification failed.*gftest.table1.*pks: (42)|(43)|(43,42)|(42,43)", result.Message)
-			ran = true
-		},
-		DataWriter: &testhelpers.MixedActionDataWriter{
-			ProbabilityOfInsert: 1.0 / 3.0,
-			ProbabilityOfUpdate: 1.0 / 3.0,
-			ProbabilityOfDelete: 1.0 / 3.0,
-			NumberOfWriters:     4,
-			Tables:              []string{"gftest.table1"},
-		},
-		Ferry: ferry,
-		DisableChecksumVerifier: true,
-	}
-
-	testcase.Run()
-	assert.True(t, ran)
-}
-
-func TestIgnoresColumns(t *testing.T) {
-	ferry := testhelpers.NewTestFerry()
-	iterativeVerifier := &ghostferry.IterativeVerifier{}
-	ran := false
-
-	testcase := &testhelpers.IntegrationTestCase{
-		T: t,
-		SetupAction: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			setupSingleTableDatabase(ferry, sourceDB, targetDB)
-			iterativeVerifier.IgnoredColumns = map[string]map[string]struct{}{"table1": {"data": struct{}{}}}
-		},
-		AfterRowCopyIsComplete: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			setupIterativeVerifierFromFerry(iterativeVerifier, ferry.Ferry)
-
-			err := iterativeVerifier.Initialize()
-			testhelpers.PanicIfError(err)
-
-			err = iterativeVerifier.VerifyBeforeCutover()
-			testhelpers.PanicIfError(err)
-		},
-		BeforeStoppingBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			ensureTestRowsAreReverified(ferry)
-		},
-		AfterStoppedBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			modifyDataColumnInSourceDB(ferry)
-
-			result, err := iterativeVerifier.VerifyDuringCutover()
-			assert.Nil(t, err)
-			assert.True(t, result.DataCorrect)
-			assert.Equal(t, "", result.Message)
-			ran = true
-		},
-		DataWriter: &testhelpers.MixedActionDataWriter{
-			ProbabilityOfInsert: 1.0 / 3.0,
-			ProbabilityOfUpdate: 1.0 / 3.0,
-			ProbabilityOfDelete: 1.0 / 3.0,
-			NumberOfWriters:     4,
-			Tables:              []string{"gftest.table1"},
-		},
-		Ferry: ferry,
-		DisableChecksumVerifier: true,
-	}
-
-	testcase.Run()
-	assert.True(t, ran)
-}
-
-func TestIgnoresTables(t *testing.T) {
```

Comment:
Not applicable. The InlineVerifier just follows the table filter as configured globally for Ghostferry. Any tables not loaded into the TableSchemaCache will not be verified.